Source/Internal/Views/VirtusizeProductImageView.swift

```diff
@@ -114,9 +114,12 @@ internal class VirtusizeProductImageView: UIView {
         })
     }
 
-    private func getCloudinaryImageUrl(_ cloudinaryPublicId: String) -> URL? {
+    private func getCloudinaryImageUrl(_ cloudinaryPublicId: String?) -> URL? {
+        if cloudinaryPublicId == nil {
```

Comment:
```suggestion
        guard let cloudinaryPublicId = cloudinaryPublicId {
          return nil
        }
        return URL(string:
              "https://res.cloudinary.com/virtusize/image/upload/t_product-large-retina-v1/\(cloudinaryPublicId).jpg")
```

Reply:
Using a guard here is more iconic in Swift. `URL(string:)` has already the type `URL?` so no need to coalesce it to `nil` manually

Reply:
@zr0z sounds good. I changed it in this commit cfeee9d